apps/site/assets/ts/components/Alerts.tsx

```diff
@@ -147,8 +147,11 @@ const alertDescription = (alert: AlertType): ReactElement<HTMLElement> => (
 const Alert = ({ alert }: { alert: AlertType }): ReactElement<HTMLElement> => {
   const [expanded, toggleExpanded] = useState(false);
   const onClick = (): void => toggleExpanded(!expanded);
+
+  const alertUrl = alert.url ? alert.url : "";
+
   // remove [http:// | https:// | www.] from alert URL:
-  let strippedAlertUrl = alert.url.replace(/(https?:\/\/)?(www\.)?/i, "");
+  let strippedAlertUrl = alertUrl.replace(/(https?:\/\/)?(www\.)?/i, "");
```

Comment:
It might already be part of a snapshot, but since this seems important would you mind please adding a test explicitly showing that this URL is displayed properly?